docs/Makefile

```diff
@@ -6,7 +6,7 @@ ifneq ($(EXAMPLES_PATTERN),)
 endif
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -j auto $(EXAMPLES_PATTERN_OPTS)
+SPHINXOPTS    = -W -j auto -T -v $(EXAMPLES_PATTERN_OPTS)
```

Comment:
nit: according to [this](https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-v) you may only need -v. But will defer to your hard-earned hands-on knowledge here

Reply:
Oh, I know we'll only need -v, but having -T be explicit seemed better (to show tracebacks)